domain-mapping/cli/class-darkmatter-domain-cli.php

```diff
@@ -383,10 +421,23 @@ public function set( $args, $assoc_args ) {
 			$active = false;
 		}
 
+		/**
+		 * Handle the Media flag.
+		 */
+		$domain_types = [
+			'main'  => DM_DOMAIN_TYPE_MAIN,
+			'media' => DM_DOMAIN_TYPE_MEDIA,
+		];
+		$type         = DM_DOMAIN_TYPE_MAIN;
+
+		if ( ! empty( $opts['type'] ) && array_key_exists( $opts['type'], $domain_types ) ) {
+			$type = $domain_types[ $opts['type'] ];
+		}
+
```

Comment:
duplicate code

Reply:
I've refactored this - and the other duplicate code - into a separate method.